motioneye/config.py

```diff
@@ -426,7 +488,9 @@ def get_camera(camera_id, as_lines=False):
         _set_default_simple_mjpeg_camera(camera_id, camera_config)
 
     else:  # incomplete configuration
-        logging.warning('camera config file at %s is incomplete, ignoring' % camera_config_path)
+        logging.warning(
+            'camera config file at %s is incomplete, ignoring' % camera_config_path
```

Comment:
I wonder why this has not become:
```py
f'camera config file at {camera_config_path} is incomplete, ignoring'
```
via #2348. Same in other cases here where the `%` variable injection is still used 🤔.

Reply:
See pyupgrade rules.

Reply:
The rule doesn't give more details about when if is skipped:
- First step: https://github.com/asottile/pyupgrade#printf-style-string-formatting
- Python 3.6+: https://github.com/asottile/pyupgrade#f-strings

Examples are only with at least two variables, but single variable strings have been converted as well, which makes sense for consistency.